L/libgifextra/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GifLibExtra"
```

Comment:
This name must match exactly the name of the directory, so either change this name or rename the directory.  I'd probably suggest to rename the directory.

Reply:
But the name of the git directory is https://github.com/ashwani-rathee/GifLibExtra , is `GifLibExtra` which is same as name here. I am sure I am missing something sorry, can you tell? I think I need to change the Git directory name to libgifextra only?and everything else too with GifLibExtra

Reply:
Quoting (again) from https://github.com/JuliaPackaging/Yggdrasil/pull/4882#discussion_r867361119

> This name _**must**_ match the name of the directory where the file is in

The directory of https://github.com/JuliaPackaging/Yggdrasil/blob/bf459f5b2f86641e0f48f027c37189da61ff96d6/L/libgifextra/build_tarballs.jl is called `L/libgifextra`

Reply:
Okay, parts of the problem solved. Thanks!!!